README.md

```diff
@@ -18,13 +18,23 @@ _Add examples here_
 
 _Add examples here_
 
-## Testing
+## Contributing
 
-Run `yarn test` to run the tests once.
+### Setup
 
-To run tests on file changes, run `yarn test:watch`.
+- Install [Node.js](https://nodejs.org) version 12
+  - If you are using [nvm](https://github.com/creationix/nvm#installation) (recommended) running `nvm use` will automatically choose the right node version for you.
+- Install [Yarn v1](https://yarnpkg.com/en/docs/install)
+- Run `yarn setup` to install dependencies and run any requried post-install scripts
+  - **Warning**: Do not use the `yarn` / `yarn install` command directly. Use `yarn setup` instead. The normal install command will skip required post-install scripts, leaving your development environment in an invalid state.
```

Comment:
```suggestion
  - **Warning:** Do not use the `yarn` / `yarn install` command directly. Use `yarn setup` instead. The normal install command will skip required post-install scripts, leaving your development environment in an invalid state.
```

Reply:
Hmm, interesting, I never know whether to bold the colon. This seems fine though.

Reply:
I think it just looks (and is conceptually) more cohesive if the entire thing is bolded, as opposed to just the word. Pick your own adventure:

**Warning**: Foo.

vs

**Warning:** Foo.

Reply:
A different angle: why _shouldn't_ the colon also be bolded?